src/cls/IPM/Main.cls

```diff
@@ -3789,4 +3825,36 @@ ClassMethod Information(ByRef pCommandInfo)
     do ..DisplayModules(.updatedList)
 }
 
+ClassMethod Update(ByRef pCommandInfo)
+{
+    set verbose = $get(pCommandInfo("data","Verbose"))
+    set version = $get(pCommandInfo("parameters","version"))
+    set path = $$$GetModifier(pCommandInfo,"path")
+    set moduleName = $get(pCommandInfo("parameters","module"))
+    set moduleCurrent = ##class(%IPM.Storage.Module).NameOpen(moduleName,.tSC)
+    $$$ThrowOnError(tSC)
+    if moduleCurrent = "" {
+        write !,"ERROR: Module "_moduleName_" has not been installed. Please use the install or load commands to install the module before calling update.",!
+        quit
+    }
+    if verbose {
+        if version = "" {
+            write !,"Resolving latest available version of module ",moduleName," to install",!
```

Comment:
Should check if path is non-empty and indicate it will just take whatever version is in that then